contrib/gammastep/template.py

```diff
@@ -0,0 +1,30 @@
+pkgname = "gammastep"
+pkgver = "2.0.9"
+pkgrel = 0
+build_style = "gnu_configure"
+make_cmd = "gmake"
+hostmakedepends = [
+    "autoconf",
+    "gettext-devel",
+    "automake",
+    "slibtool",
+    "intltool",
+    "pkgconf",
+    "wayland-progs",
+    "gmake",
+]
+makedepends = [
+    "wayland-devel",
+    "libdrm-devel",
+    "libx11-devel",
+    "libxcb-devel",
+    "glib-devel",
+    "geoclue-devel",
+    "libxxf86vm-devel",
+]
+pkgdesc = "Redshift alternative with wayland support"
+maintainer = "jabuxas <lucasbarbieri.c@gmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.com/chinstrap/gammastep"
+source = f"{url}/-/archive/v{pkgver}/gammastep-v{pkgver}.tar.bz2"
```

Comment:
```suggestion
source = f"{url}/-/archive/v{pkgver}.tar.gz"
```

shorter version of the same

also for github autotarballs .gz is preferred

Reply:
ok